hbase-client/src/main/java/org/apache/hadoop/hbase/exceptions/ClientExceptionsUtil.java

```diff
@@ -111,28 +111,6 @@ public static Throwable findException(Object exception) {
     return null;
   }
 
-  /**
-   * Checks if the exception is CallQueueTooBig exception (maybe wrapped
-   * into some RemoteException).
-   * @param t exception to check
-   * @return true if it's a CQTBE, false otherwise
-   */
-  public static boolean isCallQueueTooBigException(Throwable t) {
```

Comment:
these were not used anywhere and i usually find unused code like this confusing when reading through the code base, so decided to delete since this is an IA.Private class.